controller/rpc/pb/controller.proto

```diff
@@ -23,6 +23,11 @@ service ControllerService {
     rpc ReplicaPrepareRebuild(ReplicaAddress) returns (ReplicaPrepareRebuildReply);
     rpc ReplicaVerifyRebuild(ReplicaAddress) returns (ControllerReplica);
 
+    rpc BackupReplicaMapCreate(BackupReplicaMap) returns (google.protobuf.Empty);
+    rpc ReplicaAddressGet(ReplicaAddressRequest) returns (ReplicaAddressResponse);
+    rpc BackupGet(google.protobuf.Empty) returns (BackupGetResponse);
+    rpc BackupDelete(BackupDeleteRequest) returns (google.protobuf.Empty);
```

Comment:
It's not `delete a backup`. It deletes a backup ID from controller's backup replica address map.

`BackupReplicaMapDelete`.